docs/configuration/reference/agent.md

```
@@ -566,3 +566,22 @@ Or via environment variable:
 ```bash
 ARGOCD_AGENT_ON_APPLICATION_RECREATE=clear-status
 ```
+
+### Application Adoption
+| | |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)